diff.py

```diff
@@ -2593,6 +2670,7 @@ class Display:
     def __init__(self, basedump: str, mydump: str, config: Config) -> None:
         self.config = config
         self.base_lines = process(basedump, config)
+        post_process(self.base_lines)
```

Comment:
move this call to the end of `process`; it's not great to require the caller to call both of process and post_process for correctness

Reply:
Done.